qa/rolling-upgrade/build.gradle

```diff
@@ -80,6 +80,7 @@ for (Version bwcVersion : bwcVersions.wireCompatible) {
     doFirst {
       project.delete("${buildDir}/cluster/shared/repo/${baseName}")
     }
+    systemProperty 'tests.upgrade_from_version', "${baseName}"
```

Comment:
`baseName` is prefixed with `v`, right? I don't think that is correct, this should be `bwcVersion` which is parseable by `Version.fromString`

Reply:
good point, I was unnecessarily striping that v. bwcVersion is good enough